programs/drift/src/instructions/user.rs

```diff
@@ -402,6 +402,10 @@ pub fn handle_withdraw(
     )?;
 
     validate!(!user.is_bankrupt(), ErrorCode::UserBankrupt)?;
+    validate!(
```

Comment:
think before we were just dependent on checking initial margin below. maybe it's best to check if user has exited liq instead of blocking here?